examples/nfs-pd/testpd.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1beta1
+desiredState:
+  manifest:
+    containers:
+      - name: testpd
+        image: dockerfile/nginx
+        volumeMounts:
+            # name must match the volume name below
+            - name: nfs
+              mountPath: "/var/www/html/mount-test"
```

Comment:
Except that it doesn't seem to mount if I specify a path that conflicts with a declared Volume in the image's Dockerfile.